controllers/board-game-review-controllers.js

```diff
@@ -57,3 +57,16 @@ exports.newComment = (req, res, next) => {
         next(err)
     });
   };
+
+  exports.editReview = (req, res, next) => {
+    const id = req.params.review_id;
+    const votes = req.body
```

Comment:
This variable is called votes but its an object with a key of inc_votes, I wonder if there is a better name for it as its the information to be updated. maybe just destructuring the inc_votes key might be more readable